standard/StandardProject.cmake

```diff
@@ -18,8 +18,13 @@ include(CMakeParseArguments) # Backwards compatibility
 
 # This must be a macro since project defines scope-local variables
 # that we generally rely on being in the root context.
+# IMPORTAINT: Note that if there is no *direct* call to 'project' in the root CMakeLists.txt
+# file, CMake will automatically inject one with the default languages of C and CXX.
```

Comment:
Thanks @yeswalrus for clarifying this via the documentation at https://cmake.org/cmake/help/v3.0/command/project.html !

Only typo remaining I see here is `s/IMPORTAINT/IMPORTANT/`